panoply/datasource.py

```diff
@@ -140,7 +140,7 @@ def wrapper(*args, **kwargs):
                              'The user would have to re-authenticate',
                              traceback.format_exc())
                     # raise a non-retryable exception
-                    raise PanoplyException(
+                    raise OathValidationException(
                         'access token could not be refreshed ({})'
                         .format(str(e)), retryable=False)
```

Comment:
If you add original error, just call it like that 
```TokenValidationException (e, 'access token could not be refreshed ({})'.format(str(e)), retryable=False)```

Reply:
But in this case we will show original exception twice. First when we put as first parameter and then when when format string

Reply:
We will only store the original error as property. And will be able to reach it in our DS.